diff --git a/www/application/config/autoload.php b/www/application/config/autoload.php
index 66cb5fa..5d8982b 100644
--- a/www/application/config/autoload.php
+++ b/www/application/config/autoload.php
@@ -58,7 +58,7 @@ $autoload['packages'] = array();
 |
 |	$autoload['libraries'] = array('user_agent' => 'ua');
 */
-$autoload['libraries'] = array('session');
+$autoload['libraries'] = array('database', 'session');
 
 /*
 | -------------------------------------------------------------------
