diff --git a/www/application/config/config.php b/www/application/config/config.php
index 07d2eee..cd6385c 100644
--- a/www/application/config/config.php
+++ b/www/application/config/config.php
@@ -23,7 +23,7 @@ defined('BASEPATH') OR exit('No direct script access allowed');
 | a PHP script and you can easily do that on your own.
 |
 */
-$config['base_url'] = $_ENV['CI_BASE_URL'];
+$config['base_url'] = getenv('CI_BASE_URL');
 
 /*
 |--------------------------------------------------------------------------
@@ -213,7 +213,7 @@ $config['directory_trigger'] = 'd';
 | your log files will fill up very fast.
 |
 */
-$config['log_threshold'] = 0;
+$config['log_threshold'] = array(1, 3);
 
 /*
 |--------------------------------------------------------------------------
@@ -439,7 +439,11 @@ $config['global_xss_filtering'] = FALSE;
 | 'csrf_regenerate' = Regenerate token on every submission
 | 'csrf_exclude_uris' = Array of URIs which ignore CSRF checks
 */
-$config['csrf_protection'] = TRUE;
+if (stripos($_SERVER["REQUEST_URI"],'/PengumumanLine/webhook') === FALSE) {
+    $config['csrf_protection'] = TRUE;
+}else{ 
+    $config['csrf_protection'] 	= FALSE; 
+}
 $config['csrf_token_name'] = 'csrf_token';
 $config['csrf_cookie_name'] = 'csrf_token';
 $config['csrf_expire'] = 7200;
@@ -512,3 +516,5 @@ $config['rewrite_short_tags'] = FALSE;
 | Array:		array('10.0.1.200', '192.168.5.0/24')
 */
 $config['proxy_ips'] = '';
+
+$config['sess_save_path'] = sys_get_temp_dir();
